### PR TITLE
Feature for user profile new fields, avatar upload and template integ…

### PR DESCRIPTION
**Pull request by techaffinity**
1. We have added new fields in the user profile 
2. Template integration for user profile

https://github.com/praekelt/actionforwomen/issues/4
https://github.com/praekelt/actionforwomen/issues/9

https://github.com/praekelt/actionforwomen/issues/5 - We have validated for 11 digit canada number

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -153,41 +153,54 @@ class EditProfileForm(RegistrationForm):
...
             'about_me',
             'baby_name',
-            'date_qualifier',
-            'delivery_date',
-            'unknown_date',
-            'baby_has_been_born'
+            # 'date_qualifier',
+            # 'delivery_date',
+            # 'unknown_date',
+            'baby_has_been_born',
```
Comment (miltontony) [inline app/forms.py]: This can also be removed. Please also just delete the fields instead of commenting them out.

File: app/forms.py [hunk 2]
```diff
@@ -153,41 +153,54 @@ class EditProfileForm(RegistrationForm):
         label="Baby has been born",
         required=False
     )
+    image = forms.ImageField()
+    sur_name = forms.CharField(
```
Comment (miltontony) [inline app/forms.py]: This is not needed. You can use `user.last_name` field from the default Django user model.

File: app/forms.py (hunk 1 above)
Comment (techaffinity) [inline app/forms.py]: Do we need to remove about_me,baby_name also?
Do you want remove only from code or we have to remove from database?

Comment (miltontony) [inline app/forms.py]: You can remove from the database too

**Comment by miltontony** [comment]
:+1: 
closes #4 
closes #5 
closes #9

**Pull request merged**